assets/app/features/upgrade/steps.factory.js

```diff
@@ -10,11 +10,20 @@
         var factory = {
             steps: initialSteps(),
             activeStep: {},
-            refeshStepsList: refeshStepsList
+            refeshStepsList: refeshStepsList,
+            setCurrentStepCompleted: setCurrentStepCompleted,
+            isStepFinished: isStepFinished
```

Comment:
Maybe change this to `isCurrentStepCompleted` for consistency with `setCurrentStepCompleted`?